ONSdigital/response-operations-social-ui
# Non JS solution for cancel link on change status spoke.

### Context
The cancel link on the change response status page was using inline javascript, this was broken on some browsers. This PR changes it to link to the request referrer URL.

### What has changed
* Replaced javascript back function with request referrer link

### How to test
Go to change a response status and click the cancel link, it should still take you back to the page you came from. Try this with JS disabled, it should now still work.

### Links
https://trello.com/c/L2gGHlfU/294-chrome-on-change-response-status-screen-cancel-link-does-not-work-works-in-ie
